data/assets/scene/milkyway/constellations/big_dipper.asset

```diff
@@ -0,0 +1,58 @@
+local constellations_helper = asset.require("util/constellations_helper")
+
+
+
+local speck = asset.resource({
+  Name = "Big Dipper Constellation Files",
+  Type = "HttpSynchronization",
+  Identifier = "bigdipper_constellation_data",
+  Version = 1
+})
+
+
+local BigDipper = {
+  Identifier = "BigDipperConstellation",
+  Renderable = {
+    Type = "RenderableConstellationLines",
+    Enabled = false,
+    Labels = {
+      File = speck .. "bigdipper.label",
+      Opacity = 0.3,
+      Color = { 0.8, 0.8, 0.8 },
+      Size = 14.5,
+      MinMaxSize = { 8, 170 },
+      Unit = "pc"
+    },
+    Opacity = 0.3,
+    File = speck .. "bigdipper.speck",
+    NamesFile = speck .. "bigdipper.dat",
+    Colors = { { 0.6, 0.4, 0.4 }, { 0.8, 0.0, 0.0 }, { 0.0, 0.3, 0.8 } },
+    Unit = "pc",
+    DimInAtmosphere = true
+  },
+  Tag = { "daytime_hidden" },
+  GUI = {
+    Name = "Big Dipper",
+    Path = "/Milky Way/Constellations",
+    Description = [[Just the Big Dipper]]
```

Comment:
Should have a bit longer and more descriptive description.  Or alternatively take this away